Library/Homebrew/manpages/brew.1.md.erb

```diff
@@ -30,29 +30,53 @@ Linux distribution without requiring `sudo`.
 
 ## TERMINOLOGY
 
-**formula**: Homebrew package definition that builds from upstream sources
+**formula**
 
-**cask**: Homebrew package definition that installs macOS native applications
+: Homebrew package definition that builds from upstream sources
```

Comment:
Why is this now split on two lines?

Reply:
Because this is how the manpage has always been displayed under `ronn`:

<img width="784" alt="Screenshot 2024-03-10 at 06 13 11" src="https://github.com/Homebrew/brew/assets/1190754/ad9f4397-5050-4010-8d4b-9b07a6573aa6">

This is [definition list](https://kramdown.gettalong.org/quickref.html#definition-lists) syntax.

Bonus is now https://docs.brew.sh/Manpage will be displayed the same.

Reply:
> this is natively supported and is [definition list](https://kramdown.gettalong.org/quickref.html#definition-lists) syntax

Okay, now it makes sense to me.